### PR TITLE
Show the block supports for static blocks tutorial in the handbook

### PR DESCRIPTION
**Pull request by oandregal**
Follow up to https://github.com/WordPress/gutenberg/pull/38210

This PR aims to fix two issues:

- The link at the end of the [block controls section](https://developer.wordpress.org/block-editor/how-to-guides/block-tutorial/block-controls-toolbar-and-sidebar/) points to a non-existing page. I believe this is because it's relative and not absolute (`docs/...`  vs `/docs/...).
- The page doesn't show up in the block tutorial sidebar. I believe this has to do with another page having the same slug (block supports). This PR changes the slug so it's unique.

### CODE REVIEW

File: docs/how-to-guides/block-tutorial/block-controls-toolbar-and-sidebar.md [hunk 1]
```diff
@@ -280,4 +280,4 @@ registerBlockType( 'create-block/gutenpride', {
 Block controls rendered in both the toolbar and sidebar will also be used when
 multiple blocks of the same type are selected.
 
-**Note :** In the example above, we added text and background color customization support to our block to demonstrate the use of `InspectorControls` to add custom controls to the sidebar. That said, for common customization settings including color, border, spacing customization and more, we will see on the [next chapter](docs/how-to-guides/block-tutorial/block-supports.md) that you can rely on block supports to provide the same functionality in a more efficient way.
+**Note :** In the example above, we added text and background color customization support to our block to demonstrate the use of `InspectorControls` to add custom controls to the sidebar. That said, for common customization settings including color, border, spacing customization and more, we will see on the [next chapter](/docs/how-to-guides/block-tutorial/block-supports.md) that you can rely on block supports to provide the same functionality in a more efficient way.
```
Comment (ntsekouras) [inline docs/how-to-guides/block-tutorial/block-controls-toolbar-and-sidebar.md]:
```suggestion
**Note :** In the example above, we added text and background color customization support to our block to demonstrate the use of `InspectorControls` to add custom controls to the sidebar. That said, for common customization settings including color, border, spacing customization and more, we will see on the [next chapter](/docs/how-to-guides/block-tutorial/block-supports-in-static-blocks.md) that you can rely on block supports to provide the same functionality in a more efficient way.
```

**Review by ntsekouras — APPROVED** [review]
Thanks @oandregal !

**Comment by oandregal** [comment]
This has moved things forward: the page is now listed in the sidebar and accessible via this URL https://developer.wordpress.org/block-editor/how-to-guides/block-tutorial/block-supports-in-static-blocks/

However, the content is missing :thinking:

**Comment by oandregal** [comment]
Asking in [slack's core-editor](https://wordpress.slack.com/archives/C02QB2JS7/p1643881194997699) for advice.

**Comment by oandregal** [comment]
And now, it magically works :magic_wand:

**Pull request merged**